pkg/services/secrets/migrator/reencrypt.go

```diff
@@ -289,3 +291,90 @@ func (s alertingSecret) ReEncrypt(ctx context.Context, secretsSrv *manager.Secre
 
 	return !anyFailure
 }
+
+func (s ssoSettingsSecret) ReEncrypt(ctx context.Context, secretsSrv *manager.SecretsService, sqlStore db.DB) bool {
+	results := make([]*models.SSOSettings, 0)
+
+	err := sqlStore.WithDbSession(ctx, func(sess *db.Session) error {
+		return sess.Find(&results)
```

Comment:
I guess it would be enough to fetch the rows where `is_deleted = 0`. Wdyt?

Reply:
I think it's better to migrate all secrets from the database even for deleted entities. There is no point in keeping records in the database with secrets that cannot be decrypted.